pkg/action/install.go

```diff
@@ -149,3 +149,11 @@ func (i *Install) Chart(args []string) (string, error) {
 	// len(args) == 1
 	return args[0], nil
 }
+
+// NameAndChart overloads Helm's NameAndChart. It always fails.
+//
+// On Hypper, we need to read the chart annotations to know the correct release name.
```

Comment:
@mattfarina (et al): is this needed?  I need to "overload" this function as it its usage with Hypper libs is incorrect. Since there doesn't seem to be any other way in Golang, "it eez what it eez".